docs/_navbar.md

```diff
@@ -0,0 +1,6 @@
+* <!-- markdown-link-check-disable -->
+* [Home](/README.md)
+* [Release Notes](/release-notes.md)
+* [Support & Contribution](/support-contribution.md)
+* <!-- markdown-link-check-enable -->
```

Comment:
Adding that comment creates an additional space in the UI that does not look good. Could try diable the check differently?